crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -236,6 +236,52 @@ impl Display for DisplayGenericAlias<'_> {
     }
 }
 
+impl<'db> GenericContext<'db> {
+    pub fn display(&'db self, db: &'db dyn Db) -> DisplayGenericContext<'db> {
+        DisplayGenericContext {
+            typevars: self.variables(db),
+            db,
+        }
+    }
```

Comment:
This isn't actually used in the PR, but I _was_ using it for some `printf` debugging at one point, and it seemed worth keeping.